src/stores/b2b-features.md

```diff
@@ -41,31 +41,41 @@ By default, all B2B features are initially disabled. However, they are always av
       ![B2B configuration - enable company settings]({% link images/images-b2b/config-general-b2b-features-company-enabled.png %}){: .zoom}
       [_B2B Configuration_]({% link configuration/general/b2b-features.md %})
 
-1. To establish a default payment method for B2B orders, set **Applicable Payment Methods** to one of the following:
+1. To establish the default payment method(s) for B2B orders, set **Applicable Payment Methods** to one of the following:
 
    - All Payment Methods
 
    - Specific Payment Methods
 
       For Specific Methods, select the **Payment Methods** that you want to make available to your customers by holding down the Ctrl key (PC) or the Command key (Mac) as you click each option.
```

Comment:
if specific methods is a field name, it should be bolded.  if not, should it be capped?

Reply:
I am think not capped since that is not the title/label.